pkg/kubelet/util/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Utility functions.
+package util
```

Comment:
@mbforbes, I'm okay with changing to a different package name. "pod" is not a good name since this file may include non-pod objects in the future. I could use "format" if other reviewers agree, or simply merge it to the existing types package (which is again not very unique).

Just a side node, I need this file to be in a separate package so that we could use it for logging in other packages under the kubelet directory.